WORKSPACE

```diff
@@ -0,0 +1,304 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository", "git_repository")
+
+git_repository(
+    name = "build_bazel_rules_apple",
+    remote = "https://github.com/bazelbuild/rules_apple.git",
+    tag = "0.6.0",
+)
+
+load(
+    "@build_bazel_rules_apple//apple:repositories.bzl",
+    "apple_rules_dependencies",
+)
+
+apple_rules_dependencies()
+
+git_repository(
+    name = "build_bazel_rules_swift",
+    remote = "https://github.com/bazelbuild/rules_swift.git",
+    tag = "0.3.0",
+)
+
+load(
+    "@build_bazel_rules_swift//swift:repositories.bzl",
+    "swift_rules_dependencies",
+)
+
+swift_rules_dependencies()
+
+## SPM Dependencies
```

Comment:
Are these going to be maintained manually? You had mentioned generating their build definitions via SPM's info.

Reply:
We certainly can do that but it might be a bit more work to make sure we’re handling things like Yams correctly. Do you think it’s worth submitting PRs with BUILD files